connector/src/rest/connector/libs/tmos/.python-version

```diff
@@ -0,0 +1 @@
+net_testing
```

Comment:
I think this is because of pyvenv,  though we dont need this in the package, can you remove it ?

Reply:
will do.